auth-web/src/views/auth/home/RequestNameView.vue

```diff
@@ -60,16 +62,23 @@ import NumberedCompanyTooltip from '@/components/auth/common/NumberedCompanyTool
   }
 })
 export default class RequestNameView extends Vue {
-  private readonly nroUrl = ConfigHelper.getNroUrl()
   private readonly learnMoreUrl = 'https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/businesses-incorporated-companies/approval-business-name'
   private bulletPoints: Array<any> = [
     { text: 'Create a unique name that ensures the public is not confused or misled by similar corporate names.' },
     { text: 'Submit this name for examination by the Business Registry.' },
     { text: 'If your name is approved, you can use it to incorporate or register your business.' }
   ]
 
-  private get nameRequestStatusUrl () {
-    return `${this.nroUrl}nro.htm?_flowId=anonymous-monitor-flow&_flowExecutionKey=e1s1`
+  private get nameRequestExistingUrl () {
+    return LaunchDarklyService.getFlag(LDFlags.LinkToNewNameRequestApp)
+      ? `${ConfigHelper.getNameRequestUrl()}existing`
```

Comment:
I am assuming that, at some point in the future, this URL will open the "Existing Name Request" tab. In the meantime, NR UI has a fallback route so this works.

Reply:
interesting!

Reply:
Yep.